README.md

```diff
@@ -17,228 +29,178 @@ gem "talkable"
 
 ``` sh
 rails generate talkable:install
```

Comment:
Add a note that this is a Rails installation.  We should communicate how manual installation is done.